tools/InteractiveSetup/Main.cs

```diff
@@ -622,12 +622,13 @@ private static void SimpleQueuesSetup()
             config = new Dictionary<string, object>
             {
                 { "Directory", "" },
-                { "StorageType", "Volatile" }
```

Comment:
Volatile is default mode when not specified, it's on purpose. We can extend the setup wizard to ask, but we won't change the default because it would impact other things.

Reply:
What I do not understand is:

I started the wizard, I select to use simple storage, or simple queue, the wizard ask me the path on disk, then the wizard write that path on the configuration, but since the default is Volatile, the disk was never used. Then the user look at the configuration file and see that the file correctly contains the path, but nevertheless nothing get written. I find it confusing.

What is the purpose of asking the path to the user if then the configuration is Volatile? 

A better option would be to ask the user if they want to use in-memory (volatile) or disk.

Reply:
the setup wizard doesn't say path "on disk" :-)

> Directory where to store queue messages [current: _tmp/queues]:
> Directory where to store files [current: _tmp/files]:

SimpleStorage, SimpleQueue and SimpleVector internally use a filesystem that supports directories. The FS can be Persistent of Volatile, in both cases it uses "directories".  

By default it's a volatile FS, and the setup wizard doesn't ask, which is the part that could be improved.

Reply:
> I've made a change: Now the user can specify an empty path to explicitly choose Volatile, but if they specify a valid path, Disk configuration is used.

This won't work, even if volatile the path cannot be empty. As I mentioned, we won't change the default, but we can improve the wizard to ask.

Reply:
Ok understood, if you want I can change the wizard to explicitly ask if the storage must be volatile or disk, it should be a quick and simple change.

I noticed now that the question is about path and did not tell Path on disk :D :D :), but nevertheless path lead to think that this will be saved on disk. An explicit question about

Do you want to persist on disk or you want it to be volatile? 

Should suffice

Reply:
your call, if you have time to add the new question it would certainly be great :-)